archstrike-testing/python2-rekall/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+# Contributor: strayArch <j.a.stronz@gmail.com>
+
+buildarch=1
+
+_pkgname="rekall"
+pkgname="python2-${_pkgname}"
+pkgver="1.6.0"
+pkgrel=1
+pkgdesc="A Memory Forensic Framework"
+url="https://www.rekall-forensic.com/"
+arch=("any")
+groups=("archstrike" "archstrike-analysis" "archstrike-forensics")
+license=("GPL2")
+depends=("python2")
+makedepends=("python2-setuptools" "python2-virtualenv")
+replaces=("${pkgname}<${pkgver}")
+source=("https://pypi.python.org/packages/6e/1a/a34c540b33541c12caac3abf5052c18860701c1e36ef1048bbce86d868cc/rekall-${pkgver}.zip"
+        "https://raw.githubusercontent.com/google/rekall/master/LICENSE.txt")
+sha512sums=("ab1ab316359bbc44cdaafa7bf41acc4e0ad9771f22737016c318ed539ee65a69c61e31d3be71c9bf46b85470a2528b3568fc6372282b612c6fcea9ce1900b2a1"
+            "7d9f37a715aa7ddb47f60b26e8eea7048a01bb8b914c9190515eee93e49a2d27a87c25687f702ac57fcd33bf97a85f3446ea0ddbe42b4f8c75ca86654f6a0be7")
+
```

Comment:
technically, it doesn't --- the env symbolically links it. However, I'll add it anyway :)